### PR TITLE
Add documentation on how to build from source

### LINKED ISSUE #1617: Add documentation on how to build the Java bindings from the sources

### PR DESCRIPTION
**Pull request by samuel-rufi**
Fixes https://github.com/iotaledger/wallet.rs/issues/1617

### CODE REVIEW

File: bindings/java/README.md [hunk 1]
```diff
@@ -102,4 +100,118 @@ public class CreateAccount {
...
+- minimal Java version >= 8
+- Android Studio with NDK
+- latest stable version of Rust
+- cargo-ndk (https://github.com/bbqsrc/cargo-ndk)
+
+1. Generate the JAR:
+```
+git clone https://github.com/iotaledger/wallet.rs
+cd wallet.rs/bindings/java/lib
+.././gradlew jarWithoutNativeLibs
```
Comment (Thoralf-M) [inline bindings/java/README.md]: Do you really need to go into lib and then call ../?

File: bindings/java/README.md [hunk 2]
```diff
@@ -102,4 +100,118 @@ public class CreateAccount {
...
+
+2. You will find the built JAR in the `lib/build/libs` directory. Add it as a library to your Android project.
+
+3. Install the Android targets you want to support:
+```
+ rustup target add \
+   aarch64-linux-android \
+   armv7-linux-androideabi \
+   x86_64-linux-android \
+   i686-linux-android
```
Comment (Thoralf-M) [inline bindings/java/README.md]: Align!

Comment (samuel-rufi) [reply bindings/java/README.md]: Aligned 😸

File: bindings/java/README.md (hunk 1 above)
Comment (samuel-rufi) [reply bindings/java/README.md]: not really, changed

File: bindings/java/README.md [hunk 3]
```diff
@@ -102,4 +100,114 @@ public class CreateAccount {
...
+### Build for Android:
+
+Requirements:
+
+- minimum Java version >= 8
+- Android Studio with NDK
+- latest stable version of Rust
+- cargo-ndk (https://github.com/bbqsrc/cargo-ndk)
+
+1. Generate the JAR:
```
Comment (thibault-martinez) [inline bindings/java/README.md:118]: Put newlines after the steps, you sometimes do and sometimes don't

File: bindings/java/README.md [hunk 4]
```diff
@@ -102,4 +100,114 @@ public class CreateAccount {
 
 Now that you are up and running, you can get acquainted with the library using
 its [how-to guides](https://wiki.iota.org/shimmer/wallet.rs/how_tos/run_how_tos/) and the
-repository's [code examples](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/java/examples/src).
\ No newline at end of file
+repository's [code examples](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/java/examples/src).
+
+## Instead, build everything from scratch yourself:
+
+If you don't want to use the provided libraries and instead want to build everything yourself from scratch:
```
Comment (samuel-rufi) [inline bindings/java/README.md]:
```suggestion
If you don't like to use the provided libraries and instead want to build everything yourself from scratch:
```

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**